Tests/NewsServiceTests/Utils/NewsPublisherFactoryTests.cs

```diff
@@ -1,8 +1,37 @@
-﻿
+﻿using Interfaces;
+using NewsService;
+using NewsService.Utils;
+using NUnit.Framework;
+using CentralBankIntegration;
+using System;
 
 namespace NewsServiceTests.Utils
 {
     class NewsPublisherFactoryTests
     {
+        private NewsPublisherTypes centralBankNewsPublisherType = NewsPublisherTypes.CentralBank;
```

Comment:
We can set the value in Initialize() for more common style, but it does not matter i guess

Reply:
I am not sure about that. I think we had to initialize vatiables in initialize method